### PR TITLE
Hypnos (HZDR): New Modules

### PR DESCRIPTION
**Pull request by ax3l**
Update MPI & HDF5 Modules after today's maintainance.

### CODE REVIEW

File: etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example [hunk 1]
```diff
@@ -25,11 +25,11 @@ then
         module load cmake/3.7.2
         module load boost/1.62.0
         module load cuda/8.0
-        module load openmpi/1.8.6.kepler.cuda80
+        module load openmpi/2.1.2.kepler.cuda80
 
         # Plugins (optional)
         module load pngwriter/0.7.0
-        module load hdf5-parallel/1.8.15 libsplash/1.6.0
+        module load hdf5-parallel/1.8.20 libsplash/1.6.0
```
Comment (PrometheusPi) [inline etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example]: QUESTION:
have you tested libspalash/1.6.0 with hdf5-parallel/1.8.20. According to the admins comment, only libsplash/1.7.0 should work.

Comment (ax3l) [reply etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example]: I am testing it right now, let's see

Comment (ax3l) [reply etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example]: works

**Comment by ax3l** [comment]
I updated the compile suite already to include libSplash 1.7.0, but the PIConGPU CMake scripts are not forward-compatible to that version. Will still merge this manually here and that PR must be next.

**Comment by ax3l** [comment]
@PrometheusPi can you please approve & merge?

**Comment by PrometheusPi** [comment]
I will merge despite the failed test status as approved by @ax3l

File: etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example [hunk 2]
```diff
@@ -25,11 +25,11 @@ then
         module load cmake/3.7.2
         module load boost/1.62.0
         module load cuda/8.0
-        module load openmpi/1.8.6.kepler.cuda80
+        module load openmpi/2.1.2.kepler.cuda80
```
[the remark this answers is not in the document]
Comment (PrometheusPi) [reply etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example]: Yes, I tested it without the "kepler" - I will provide a fix

**Pull request merged**